refinery.service

```diff
@@ -3,7 +3,7 @@ Description=Refinery Honeycomb Trace-Aware Sampling Proxy
 After=network.target
 
 [Service]
-ExecStart=/usr/bin/refinery -c /etc/refinery/refinery.toml -r /etc/refinery/rules.toml
+ExecStart=/usr/bin/refinery -c /etc/refinery/refinery.yaml -r /etc/refinery/rules.yaml
```

Comment:
Is there a way to load this like so?

* Load either the toml or yaml file, depending on which is present
* Error if both are present, ask to pick just one

Reply:
This would require changing the way refinery loads configs, or bootstrapping all this with a script to check for file existence.

Reply:
changing both `refinery.service` and `refinery.upstart` like this seems like it would be a breaking change to me